decidim-accountability/lib/decidim/accountability/admin_engine.rb

```diff
@@ -14,10 +14,9 @@ class AdminEngine < ::Rails::Engine
       routes do
         resources :statuses
         resources :results, except: [:show] do
+          get :proposals_picker, on: :collection
```

Comment:
Mmm, it is true that the endpoint serves only the proposals list when filtering proposals, but it is also used to render the whole picker for the initial load. So, I'd be ok with the name changing if we only use it for the proposals list, but I think that it could be confusing with the current behavior. What do you think?

Reply:
ok, it also serves the rendering of the picker. Let's keep it as it is :+1: